x-pack/plugins/watcher/__fixtures__/get_watch_history.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment, { Moment } from 'moment';
+import { ACTION_STATES, WATCH_STATES } from '../common/constants';
+import { ClientWatchStatusModel, ClientActionStatusModel } from '../common/types';
+
+interface WatchHistory {
+  id: string;
+  watchId: string;
+  startTime: Moment;
+  watchStatus: {
+    state: ClientWatchStatusModel['state'];
+    comment?: string;
+    lastExecution: Moment;
+    actionStatuses?: Array<{
+      id: string;
+      state: ClientActionStatusModel['state'];
+    }>;
+  };
+  details?: object;
+}
+
+export const getWatchHistory = ({
```

Comment:
I just renamed this file to align with the exported function name and reduced the function's params.